src/openvpn/tun.c

```diff
@@ -1169,15 +1174,50 @@ do_ifconfig_ipv6(struct tuntap *tt, const char *ifname, int tun_mtu,
         openvpn_execve_check(&argv, es, 0, "Solaris ifconfig IPv6 mtu failed");
     }
 #elif defined(TARGET_OPENBSD) || defined(TARGET_NETBSD) \
-    || defined(TARGET_DARWIN) || defined(TARGET_FREEBSD) \
-    || defined(TARGET_DRAGONFLY)
+    || defined(TARGET_DARWIN) || defined(TARGET_DRAGONFLY)
     argv_printf(&argv, "%s %s inet6 %s/%d mtu %d up", IFCONFIG_PATH, ifname,
                 ifconfig_ipv6_local, tt->netbits_ipv6, tun_mtu);
     argv_msg(M_INFO, &argv);
 
     openvpn_execve_check(&argv, es, S_FATAL,
                          "generic BSD ifconfig inet6 failed");
 
+#if defined(TARGET_OPENBSD) || defined(TARGET_NETBSD) \
```

Comment:
You need to move the code, not copy it. But I can't suggest that part via the GitHub UI since that is too far outside the code changes ;)